### PR TITLE
fix Kestrel version to fix build

### PR DESCRIPTION
**Pull request by analogrelay**
Also added some logging that was useful in figuring out the issue.

This whole "old versions of Kestrel break stuff" thing is getting a little annoying :)

### CODE REVIEW

File: samples/EchoApp/Properties/launchSettings.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:3159/",
+      "applicationUrl": "http://localhost:62225/",
```
Comment (Tratcher) [inline samples/EchoApp/Properties/launchSettings.json]: ?

**Comment by pakrym** [comment]
@natemcmaster @ryanbrandenburg shouldn't this be happening automatically? Do we need to look at updaterepos configuraion?

**Review by pakrym — APPROVED** [review]

File: test/Microsoft.AspNetCore.WebSockets.Test/WebSocketMiddlewareTests.cs [hunk 2]
```diff
@@ -90,24 +105,27 @@ public class WebSocketMiddlewareTests
...
-                var serverBuffer = new byte[orriginalData.Length];
-                var result = await webSocket.ReceiveAsync(new ArraySegment<byte>(serverBuffer), CancellationToken.None);
-                Assert.True(result.EndOfMessage);
-                Assert.Equal(orriginalData.Length, result.Count);
-                Assert.Equal(WebSocketMessageType.Binary, result.MessageType);
-                Assert.Equal(orriginalData, serverBuffer);
-            }))
-            {
-                using (var client = new ClientWebSocket())
+                var orriginalData = Encoding.UTF8.GetBytes("Hello World");
```
Comment (jkotalik) [inline test/Microsoft.AspNetCore.WebSockets.Test/WebSocketMiddlewareTests.cs]: nit "originalData" and all other instances.

**Review by jkotalik — APPROVED** [review]

**Comment by natemcmaster** [comment]
http://aspnetci/viewType.html?buildTypeId=Coherence_UpdateRepos hasn't run since we branched for preview1. Also, it's never had a completely successful run.

**Pull request merged**